### PR TITLE
JAVA-8435: reducing logging for tutorials-integration job

### PR DESCRIPTION
**Pull request by chaos2418**

### CODE REVIEW

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/audit/EnversFooBarAuditIntegrationTest.java [hunk 1]
```diff
@@ -123,24 +123,39 @@ public final void whenFooBarsModified_thenFooBarsAudited() {
...
-        assertEquals("BAR", barRevisionList.get(0).getName());
-        assertEquals("BAR", barRevisionList.get(1).getName());
-        assertEquals("BAR1", barRevisionList.get(2).getName());
-        assertEquals("BAR1", barRevisionList.get(3).getName());
-
-        assertEquals(1, barRevisionList.get(0).getFooSet().size());
-        assertEquals(2, barRevisionList.get(1).getFooSet().size());
-        assertEquals(2, barRevisionList.get(2).getFooSet().size());
-        assertEquals(3, barRevisionList.get(3).getFooSet().size());
+        assertEquals("BAR", barRevisionList.get(0)
```
Comment (kwoyke) [inline persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/audit/EnversFooBarAuditIntegrationTest.java:126]: I think we should keep the original formatting here.

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooFixtures.java [hunk 2]
```diff
@@ -36,26 +41,30 @@ public void createBars() {
...
                 final Foo foo2 = new Foo(null);
-                if (i % 2 == 0)
+                if (i % 2 == 0) {
                     foo2.setName("LuckyFoo" + (i + 120));
+                }
                 foo2.setBar(bar);
                 session.save(foo2);
-                bar.getFooSet().add(foo);
-                bar.getFooSet().add(foo2);
+                bar.getFooSet()
```
Comment (kwoyke) [inline persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooFixtures.java:49]: Keep the one-line format here as well

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/spring/data/jpa/query/UserRepositoryIntegrationTest.java [hunk 3]
```diff
@@ -40,46 +40,42 @@ public void whenFindAllActiveUsersNativeThenAllActiveFound() {
...
+        assertThat(allUsersSortedByName).extracting("name")
                 .containsSequence("Bob", "John", "Cindy");
     }
 
     @Test
     public void whenFindAllUsersWithPaginationThenPaginated() {
         Page<User> page = userRepository.findAllUsersWithPagination(PageRequest.of(0, 1));
-        assertThat(page.stream().map(User::getId))
-                .hasSize(1)
+        assertThat(page.stream()
```
Comment (kwoyke) [inline persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/spring/data/jpa/query/UserRepositoryIntegrationTest.java:69]: The original formatting is more readable I think. Let's keep it.

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooSortingPersistenceIntegrationTest.java [hunk 4]
```diff
@@ -46,7 +50,8 @@ public void before() {
 
     @After
     public void after() {
-        session.getTransaction().commit();
+        session.getTransaction()
```
Comment (kwoyke) [inline persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooSortingPersistenceIntegrationTest.java:53]: Keep the one-liner here pls.

File: persistence-modules/spring-data-jpa-query/src/test/java/com/baeldung/exists/CarRepositoryIntegrationTest.java [hunk 5]
```diff
@@ -19,6 +20,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = {Application.class})
+@TestPropertySource(properties = {"spring.jpa.show-sql=false"})
```
Comment (kwoyke) [inline persistence-modules/spring-data-jpa-query/src/test/java/com/baeldung/exists/CarRepositoryIntegrationTest.java:23]: Wouldn't it be better to set this property in the application(-test).properties?

**Comment by kwoyke** [comment]
retest this please

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/audit/EnversFooBarAuditIntegrationTest.java (hunk 1 above)
Comment (chaos2418) [reply persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/audit/EnversFooBarAuditIntegrationTest.java]: done.

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooFixtures.java (hunk 2 above)
Comment (chaos2418) [reply persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooFixtures.java]: reverted this as well.

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/spring/data/jpa/query/UserRepositoryIntegrationTest.java (hunk 3 above)
Comment (chaos2418) [reply persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/spring/data/jpa/query/UserRepositoryIntegrationTest.java]: done

File: persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooSortingPersistenceIntegrationTest.java (hunk 4 above)
Comment (chaos2418) [reply persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/hibernate/FooSortingPersistenceIntegrationTest.java]: yep done.

File: persistence-modules/spring-data-jpa-query/src/test/java/com/baeldung/exists/CarRepositoryIntegrationTest.java (hunk 5 above)
Comment (chaos2418) [reply persistence-modules/spring-data-jpa-query/src/test/java/com/baeldung/exists/CarRepositoryIntegrationTest.java]: I've removed the property from individual tests and added it to application.properties.

**Comment by chaos2418** [comment]
retest this please

**Pull request merged**